src/arch/x86_64/Encoding.zig

```diff
@@ -767,7 +767,7 @@ fn estimateInstructionLength(prefix: Prefix, encoding: Encoding, ops: []const Op
 }
 
 const mnemonic_to_encodings_map = init: {
-    @setEvalBranchQuota(30_000);
+    @setEvalBranchQuota(50_000);
```

Comment:
I saw you ask if this could be changed to an unstable sort, but I did not see the response. What was the result of that discussion?

Reply:
none of algorithms improved this so i started increasing it until i reached 50k. maybe we need to invent a comptime friendly sort for this usecase?
(btw i deleted that message lol)